<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USER-PC" name="Test" tests="11" failures="2" timestamp="5 Feb 2022 08:21:07 GMT" time="0.141" errors="0">
  <testcase name="tc1" time="0.031" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc1 -->
  <testcase name="tc1" time="0.0" classname="org.test.ExamplFailClass">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.test.ExamplFailClass.tc1(ExamplFailClass.java:11)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase name="tc2" time="0.0" classname="org.test.ExamplFailClass"/>
  <testcase name="tc3" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.016" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.0" classname="org.test.ExamplFailClass">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.0" classname="org.test.ExamplFailClass">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.test.ExamplFailClass.tc3(ExamplFailClass.java:23)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
</testsuite> <!-- Test -->
